web/src/components/core/Popup.jsx

```diff
@@ -214,7 +217,7 @@ const Popup = ({
       actions={actions}
       className={`${className} block-size-${blockSize} inline-size-${inlineSize}`.trim()}
     >
-      {content}
+      {isLoading ? <Loading text={_("Loading data...")} /> : content}
```

Comment:
By any means I want to make your PR more longer than needed. But what do you think about taking the opportunity for adding the "isLoadingText" prop to make possible customize it when needed. Maybe there is a place when we want to be explicit about what data is the dialog loading. E.g., "Loading available devices...". Or to add more information, "Loading encryption data. Be patient, this can take a while".

Please, feel free to not address it here since I fully understand is completely out of topic.

Reply:
BTW, I meant a prop with a default value, of course `isLoadingText = _("Loading data...")`